src/pulse-labs-recorder.ts

```diff
@@ -66,14 +67,31 @@ class PulseLabsRecorder {
     return this.sendDataToServer(requestBody, response, this.getIntegrationType());
   }
 
-  private sendDataToServer(request:any, response: any, integrationType:string) {
+  /**
+   * This method is called to send request and response
+   * data to the pulselabs server for google action.
+   * @param conv -> a DialogFlowConversation object
+   */
+  configureActionLogger(conv: DialogflowConversation) {
+    const pulseSerialize = conv.serialize;
+    conv.serialize = () => {
+      const response = pulseSerialize.call(conv);
+      this.sendDataToServer(conv.request, response, this.getIntegrationType(), 'google');
```

Comment:
The integration type should be "GOOGLE_SDK=googleSDK"

Reply:
Also you should send `conv.body` as the request. This matches with the format mentioned at https://developers.google.com/assistant/actions/build/json/dialogflow-webhook-json#simple-conversation-request-example-df